tasks/scripts/containerd-integration

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e -u
+
+export GOPATH=$PWD/gopath
+export PATH=$GOPATH/bin:$PATH
+
+source ci/tasks/scripts/docker-helpers.sh
+
+main() {
+  sanitize_cgroups
+  start_containerd
+  run_test
+}
+
+start_containerd() {
+  mkdir -p /scratch/containerd
+
+  echo '
+disabled_plugins = ["cri"]
+root = "/scratch/containerd"
+state = "/run/containerd"
+
+[grpc]
+  address = "/run/containerd/containerd.sock"
+
+[debug]
+  address = "/run/containerd/debug.sock"
+  level = "debug"
+
+[plugins]
+  [plugins."io.containerd.runtime.v1.linux"]
+    runtime = "runc"
+    shim = "containerd-shim"
+' >./containerd.toml
```

Comment:
is it maybe worth tracking this config in a separate file? I could see it going into `deployments/`, which also holds our terraform config for bin-smoke. I just realized that the README is wrong to describe it as containing BOSH-related configuration.

Reply:
> is it maybe worth tracking this config in a separate file?

yeahh, I think it makes sense - just how for `smoke` we have the `systemd` service configuration there 👍  

>  I just realized that the README is wrong to describe it as containing BOSH-related configuration.

thaaat's indeed wrong  haaha